MTM-46159 support user access requirements (#1458)
SoftwareAG/c8y-docs: content/enterprise-tenant/support-user-access.md

  - platform_administration
---

{{< c8y-admon-req >}}

> For a support user to log in as a user of your tenant, **support user access** must be enabled. This option is available at the level of each tenant and applies to access for all of its users.

Reviewer: ```suggestion
To allow support users to log in as a user of your tenant, **support user access** must be enabled. This option is available at tenant level and applies to all users of the tenant.
```

Author: yes - i've changed it
